example.php

```diff
@@ -5,7 +5,10 @@
 use Buffer\BuffLog\Bufflog;
 
 // putenv("LOG_VERBOSITY=WARNING");
-BuffLog::debug("I am a debug");
+BuffLog::debug("I am a debug string", 2, 2);
```

Comment:
I'm not sure about including these malformed examples - we don't want to confuse engineers about what the correct usage is. I think it makes sense to take them out and only show the 'right way' to call Bufflog?

Reply:
yes agree, this should be moved into a test instead.